_overviews/scala3-macros/tutorial/reflection.md

```diff
@@ -75,6 +106,33 @@ def g[T: Type](using Quotes) =
   ...
 ```
 
+## Symbols
+
+The APIs of `Term` and `TypeRepr` are relatively *closed* in the sense that methods produce and accept values
+whose types are defined in the API. You might notice however the presence of `Symbol`s which identify definitions.
+
+Both `Term` or `TypeRepr` (therefore `Expr` and `Type`) have an associated symbol.
+`Symbol`s make it possible to compare two definitions using `==` to know if they are the same.
+In addition `Symbol` exposes and is used by many useful methods. For example:
+
+ - `declaredFields` and `declaredMethods` allow you to iterate on the fields and members defined inside a symbol
+ - `flags` allows you to check multiple properties of a symbol
+ - `companionObject` and `companionModule` provide a way to jump to and from the companion object/class.
+ - `TypeRepr.baseClasses` returns the list of symbols of classes extended by a type. 
+ - `Symbol.pos` gives you access to the position where the symbol is defined, the source code of the definition
+ and even the filename where the symbol is defined.
+ - and many useful others that you can find in [`SymbolMethods`](https://dotty.epfl.ch/api/scala/quoted/Quotes$reflectModule$SymbolMethods.html)
+
+### To Symbol and back
+
+ - `TypeRepr.typeSymbol` returns the symbol of the type represented by `TypeRepr`. The recommended way to obtain a `Symbol` given a `Type[T]` is `TypeRepr.of[T].typeSymbol`
+ - For a singleton type, `TypeRepr.termSymbol` returns the symbol of the underlying object or value.
+ - `TypeRepr.memberType(symbol)` returns the `TypeRepr` of the provided symbol
```

Comment:
I think it’s a bit confusing that we use the same syntax `TypeRepr.foo` to refer to both methods of the companion object `TypeRepr`, and method of the type `TypeRepr`. We should use different syntaxes.

Reply:
Do you prefer that I introduce an object of type `TypeRepr` or use `TypeReprMethods.termSymbol` / `TypeReprModule.of` ?

Reply:
What about “The method `memberType(symbol)` on type `TypeRepr` …”?